README.md

```diff
@@ -194,17 +194,19 @@ specified target's database and downloads it to `tmp/`.
 `geordi dump staging -l` (with a Capistrano deploy target and the `--load`
 option) sources the dump into the development database after downloading it.
 
-If you are using multiple databases per environment, Geordi defaults to the
-"primary" database, or the first entry in database.yml. To dump a specific
-database, pass the database name like this:
+If you are using multiple databases per environment, you have to specify the
+database name. To dump a specific database, pass the database name with the
+`-d`/`--database` option. To load a dump into a specific database, pass the
+database name with the `-L`/`--load-database` option.
 
-    geordi dump -d primary
-
-Loading a dump into one of multiple local databases is not supported yet.
+For dumping, geordi defaults to the "primary" database, or the first entry in
+ database.yml. For dumping, these defaults are deactivated to avoid unwanted
+database wipes.
 
 **Options**
 - `-l, [--load=[DUMP_FILE]]`: Load a dump
-- `-d, [--database=NAME]`: Database name, if there are multiple databases
+- `-d, [--database=NAME]`: Database to read, if there are multiple databases
+- `-L, [--load-database=NAME]`: Database to write to, if there are multiple databases
```

Comment:
Would it be possible to have just one argument `--database` and skip the support for mixed dumping and sourcing e.g. "dumps staging primary" and "load into secondary"? I think we have the same number of databases on the server and local, so it would be always "primary => primary", "secondary => secondary", and so on.

This would reduce complexity on the code side and on the user side.

Reply:
Thanks for the "full-featured" implementation, @niklas-hasselmeyer. Maybe it is just a little bit too mighty – #173 could be fixed by simply adding support for reading local multi-db config during dump loading, and always using the "primary" config. 

How we'll continue from here depends on the specific requirements. I'll retrieve them and comment again when I have them. Please pause this PR until it is clear where we'll go.

Reply:
We have decided to go with the sketch of @denzelem: 
* There should be no new option `load-database`
* The `database` option should always be respected, i.e.
  * `geordi dump -d example` should dump the local "example" database
  * `geordi dump staging -d example` should dump the remote "example" database
  * `geordi dump staging -d example -d` should dump the remote "example" database and source it into the local "example" database. If either does not exist, an error message should be printed. 
  * `geordi -l tmp/staging.dump -d example` should source the given dump file into the local "example" database